### ISSUE TITLE
Non-numeric values for primary metric cannot be visualized in a 2D/3D/Parallel coordinates chart.

### ISSUE THREAD

**Issue by drabastomek** [Azure/MachineLearningNotebooks#786]
When trying to run this notebook: https://github.com/Azure/MachineLearningNotebooks/blob/master/how-to-use-azureml/ml-frameworks/scikit-learn/training/train-hyperparameter-tune-deploy-with-sklearn/train-hyperparameter-tune-deploy-with-sklearn.ipynb

we've seen the error as in the title when running the `RunDetails(hyperdrive_run).show()`. We can see two charts but with no values.

**Comment by zronaghi** [comment]
Thanks Tom, I'm able to see two charts with outputs: 1. Primary metric 'Accuracy' for each child run, 2. `Accuracy` for each parameter `kernel` or `penalty`. 

However still seeing the error mentioned after each run. Not sure if it's an error or warning, and whether the charts look correct. I've attached screenshots of the notebook.

![RunDetails](https://user-images.githubusercontent.com/20173354/74178351-0cb3a700-4c01-11ea-94f3-a753adecdd22.JPG)

![Accuracy](https://user-images.githubusercontent.com/20173354/74178370-16d5a580-4c01-11ea-9d7c-0ba7e0727518.JPG)

![AccuracyvsKernel](https://user-images.githubusercontent.com/20173354/74178376-1937ff80-4c01-11ea-8508-9ba756912113.JPG)

**Comment by swanderz** [comment]
@johnwu0604 @luzhang06 not explicitly AML studio, but related to our earlier convo about metrics display.

**Comment by zronaghi** [comment]
Any updates on this? We're looking into using a similar notebook as demo and would be great if we could suppress these warnings

**Comment by swatig007** [comment]
@JennaEllis @dawei li, do we still see this issue with the warning?

**Comment by JennaEllis** [comment]
This issue has been resolved as of 6/1 -closing this thread
